package.json

```diff
@@ -54,10 +54,12 @@
   "scripts": {
     "analyze": "NODE_ENV=production ANALYZE=1 node --max_old_space_size=4096 node_modules/.bin/webpack --config webpack.prod.config.js --progress --colors",
     "build": "webpack --config webpack.prod.config.js",
+    "build-dev": "webpack --config webpack.dev.config.js",
     "build-travis": "cross-env NODE_ENV=production node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --config webpack.prod.config.js",
     "heroku-postbuild": "npm run build",
     "lint": "eslint --ext .js,.ts,.tsx ./src ./test",
-    "lint.andFix": "npm run lint -- --fix",
+    "lint-andFix": "npm run lint -- --fix",
+    "postinstall": "rimraf ./src/**/*.scss.d.ts",
```

Comment:
Will this solve the problem in all cases? I encountered the problem myself, and this was not because I re-installed the package, it was because I checkout out a different branch....

Reply:
Hmm, good question.  In some cases a different branch requires doing an `npm install`, but in some cases not.  Theoretically in that case then one can run `npm run postinstall`.  But if there is a preference to make this happen on every build, which was the original plan, then I can do that.  Let me know.